### PR TITLE
Use sx props for styling

### LINKED ISSUE #1783: Update Styling System to Use sx Prop in Material-UI Components in Tablet.js
### Overview
Our project currently utilizes the makeStyles function for styling Material-UI components. However, with the updates in Material-UI version 5, it is recommended to transition to using the sx prop for styling. 
Additionally, once we don't use the makeStyles function, we will be able to remove the `mui/styles` dependency, which has conflicts with React v.18 recently added to our project. 
This GitHub issue is part of a series of Issues aiming to update the styling system in our project. In this issue, we are going to update Update Styling System to Use sx Prop in Material-UI Components in `components/FoodSeeker/SearchResults/layouts/Tablet.js` 

### Action Items
- [ ] Replace HTML elements with Material Ui components
- [ ] Replace the existing makeStyles implementation with the xs prop for styling.
- [ ] Ensure that the changes do not impact the styling overrides in the theme folder.
- [ ] Make sure you don't lose accessibility when replacing semantic HTML for Material UI Components

### Resources/Instructions
https://mui.com/material-ui/getting-started/overview/
https://mui.com/material-ui/migration/migration-v4/

### PR DESCRIPTION
**Pull request by jenny-alexander**
Fixes #1783

### CODE REVIEW

File: client/src/components/FoodSeeker/SearchResults/layouts/Tablet.js [hunk 1]
```diff
@@ -1,30 +1,14 @@
...
   return (
     <>
       {filters}
-      <div className={classes.container}>
-        <div className={classes.map}>{map}</div>
-        <div className={classes.list}>{list}</div>
-      </div>
+      <Box sx={{ height: "100%", overflow: "hidden"}}>
+        <Box sx={{ height: "50%"}}>{map}</Box>
+        <Box sx={{ height: "50%", overflow: "hidden" }}>{list}</Box>
```
Comment (SAUMILDHANKAR) [inline client/src/components/FoodSeeker/SearchResults/layouts/Tablet.js:10]: Hi @jenny-alexander, thanks for working on this issue. Quick question: I noticed that you have changed the overflow property for list from "auto" to "hidden". Visually it looks the same. Is there a reason to prefer one over the other in this case? I will read up more on it but please let me know if you have come across anything already. Thank you.

Comment (jenny-alexander) [reply client/src/components/FoodSeeker/SearchResults/layouts/Tablet.js]: @SAUMILDHANKAR - thank you for catching this typo! I fixed it. Can you review the change when you are able?

Comment (jenny-alexander) [reply client/src/components/FoodSeeker/SearchResults/layouts/Tablet.js]: Rebase done!

**Review by junjun107 — APPROVED** [review]
LGTM

**Pull request merged**